doc/contributing/security-release-process.md

```diff
@@ -190,6 +190,131 @@ out a better way, forward the email you receive to
   [Security release stewards](https://github.com/nodejs/node/blob/HEAD/doc/contributing/security-release-process.md#security-release-stewards).
   If necessary add the next rotation of the steward rotation.
 
+***
+
+## Creating proposal
+
+The process to create proposal for a Security Release is very similar to the
+common release lines. The major difference is the base repository is [`node-private`](https://github.com/nodejs-private/node-private)
+instead of [`node`](https://github.com/nodejs/node).
+
+Once the [previous steps](#planning) were completed, you should be able to
+create a proposal against `node-private`.
+
+### Step 1 - Sync vXX.x-staging branch
+
+The first step is to make sure that the
+`origin`[\[1\]](#glossary)/`vXX.x-staging`[\[2\]](#glossary) branch is updated:
+
+```console
+git remote -v
+origin  git@github.com:nodejs-private/node-private.git (fetch)
+origin  git@github.com:nodejs-private/node-private.git (push)
+upstream        git@github.com:nodejs/node.git (fetch)
+upstream        git@github.com:nodejs/node.git (push)
+
+git remote update upstream
+git reset --hard upstream/vXX.x
+```
```

Comment:
````suggestion
```console
$ git remote -v
origin  git@github.com:nodejs-private/node-private.git (fetch)
origin  git@github.com:nodejs-private/node-private.git (push)
upstream        git@github.com:nodejs/node.git (fetch)
upstream        git@github.com:nodejs/node.git (push)
$ git remote update upstream
$ git reset --hard upstream/vXX.x
```
````
In `console` codeblocks it's recommended to start command lines with `$`, `%`, `#`, etc etc, to highlight the actual command lines from the output.